.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "charts/spire"]
+	path = charts/spire
+	url = https://github.com/spiffe/helm-charts-hardened.git
```

Comment:
Will we end up with charts/spire/charts/spire/charts/spire-server? and if we try to deploy cofide/spire, will we get the top spire directory which isn't a chart?

Reply:
Also we should pin to a released tag.